addon/model.js

```diff
@@ -290,12 +289,17 @@ export default class MegamorphicModel extends EmberObject {
   }
 
   rollbackAttributes() {
+    if (DEBUG) {
+      assertNoChanges(this._store);
```

Comment:
I like this.

Reply:
yeah in general we could use a more aggressive assertions in the codebase